pkg/gui/confirmation_panel.go

```diff
@@ -248,6 +248,12 @@ func (gui *Gui) setKeyBindings(opts types.CreatePopupPanelOpts) error {
 				return nil
 			},
 		},
+		{
```

Comment:
can we move this under line 239 given it's the same action as the keybinding above?

Reply:
Like this?